### ISSUE TITLE
[Bug]: Webcontents gets wrong coordinates during ContextMenu event on touch screen.

### ISSUE THREAD

**Issue by KaminoRyo** [electron/electron#30103]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

13.1.6

### What operating system are you using?

Windows

### Operating System Version

Windows10 21H1

### What arch are you using?

x64

### Last Known Working Electron version

13.1.6

### Expected Behavior
I created a context menu using webcontents.on ("contextmenu").
No arguments other than window are passed to the popup method of menu. 
If I tap and hold, a context menu will appear at that coordinate.

### Actual Behavior
Right mouse click works, but long tap on the touch screen doesn't work. 
The context menu is displayed at incorrect coordinates.
Accurate coordinates can be obtained by clicking or context menu event in BrowserWindow.
(See the log in the Developer Tool) 

![contextMenu](https://user-images.githubusercontent.com/67363864/125380275-e8975f00-e3cc-11eb-9ad0-52bcd3d1e561.png)

### Additional Information

I noticed when testing the .exe with the Surface Pro 3. (desktop mode)

**Comment by jkleinsc** [comment]
Thanks for reporting this and helping to make Electron better!

Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a [gist](https://gist.github.com) that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/need-repro` label for this reason. After you make a test case, please link to it in a followup comment.

Thanks in advance! Your help is appreciated.

**Comment by KaminoRyo** [comment]
I create gist. You can test for bug using Electron Fiddle. 
https://gist.github.com/1773aa17d611b4d7ba4c42b6911b922a

And Every time I started Electron, I noticed that the coordinates of the Context Menu were different. 
Ummm....

**Comment by KaminoRyo** [comment]
I was able to confirm it on Surface Go 2.(tablet mode off) 
Random coordinates are determined every time Electron is started, and it seems that the context menu appears at that location regardless of the hold location.

**Comment by KaminoRyo** [comment]
**Workaround**
1. [renderer] Catch the context menu event in the window.
2. [renderer] Stop the event and pass pageX and Y to the main process through the ContextBridge.
(I needed to display "Copy image", so I parsed the element I clicked and passed it only when it was an IMG tag. )
3. [main]Catch it with ipcMain.handle() and pass the coordinates to the Menu.popup()

Reference: https://www.electronjs.org/docs/api/menu#render-process

But window.webcontents.on("'context-menu")  approach has two advantages.
1. No need to edit Renderer (important if the person who creates the web app and the person who wraps it in Electron are different)
2. You don't have to create an API with ContextBridge.

I learned this approach from the [sindresorhus/electron-context-menu](https://github.com/sindresorhus/electron-context-menu) module. I hope it will be fixed.
